packages/darklang/stdlib/fun.dark

```diff
@@ -0,0 +1,4 @@
+module Darklang =
+  module Stdlib =
+    module Fun =
```

Comment:
new blog post: Dark's standard library now includes Fun!

Reply:
Actaully, a blog post on the changes to the standard library would be cool (separate from the language changes)